src/client/src/notifications.openfin.ts

```diff
@@ -51,6 +61,27 @@ const sendFxTradeNotification = (executionTrade: ExecutionTrade) => {
   })
 }
 
+const sendQuoteAcceptedNotification = ({ rfq, quote }: RfqWithQuote) => {
+  const notification = {
+    ...rfq,
+  }
+
+  create({
+    title: `Quote Accepted: ID ${notification.id}`,
+    body: `${notification.direction} ${notification.quantity} ${notification.instrument?.name} @ ${quote?.price}`,
+    icon,
+    customData: { tradeId: notification.id },
+    buttons: [
+      {
+        title: "Highlight trade in blotter",
+        iconUrl: icon,
+        onClick: { task: TASK_HIGHLIGHT_CREDIT_TRADE },
+      },
+    ],
+    category: "Trade Executed",
+  })
```

Comment:
for the color, i am not sure there is a situation where a quote is rejected, so if it can only be accepted then the color can just be `Indicator.GREEN`

Reply:
I think I'll leave this for now, until Ed gives us some feedback, then if you could take this on as part of your ticket that would be great